packages/input/src/Input.tsx

```diff
@@ -36,6 +36,7 @@ export const PureInput = React.forwardRef<HTMLInputElement, IInputProps>(({
   value,
   onFocus,
   onBlur,
+  isArrowsHidden = false,
```

Comment:
I think the arrow should be hidden by default. And prop name can be called like showArrows.